cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -123,6 +124,14 @@ impl snowbridge_pallet_inbound_queue::Config for Runtime {
 	type AssetTransactor = <xcm_config::XcmConfig as xcm_executor::Config>::AssetTransactor;
 }
 
+pub struct AggregateMessageOriginConverter;
+
+impl Convert<H256, AggregateMessageOrigin> for AggregateMessageOriginConverter {
+	fn convert(origin: H256) -> AggregateMessageOrigin {
+		AggregateMessageOrigin::SnowbridgeV2(origin)
+	}
+}
```

Comment:
We've a shared runtime module, [snowbridge-runtime-common ](https://github.com/paritytech/polkadot-sdk/tree/master/bridges/snowbridge/runtime/runtime-common) which contains common logic that can be reused across different runtimes. Maybe it’s better to move it there?